docs/Command-Line-Interface.md

```diff
@@ -55,7 +59,8 @@ Or, if no header is provided, the default hostname and port (set in the env file
   <tr>
     <td>free</td>
     <td>free "path"</td>
-    <td>Free a file or all files under a directory from memory. File/directory will still be available in underfs.</td>
+    <td>Free a file or all files under a directory from memory. File/directory will still be
```

Comment:
"from Tachyon space" instead of from "memory" since it may be in ssd in tiered storage. 

"If the file/directory is also in under storage, it will still be available there."